test/storages.test.ts

```diff
@@ -0,0 +1,35 @@
+import { createStorageContentSignature } from '@apify/utilities';
+
+describe('createStorageContentSignature()', () => {
+    it('should set expiresAt to 0 for a non-expiring signature', () => {
+        const secretKey = 'hmac-secret-key';
+        const message = 'resource-id';
+
+        const signature = createStorageContentSignature({
+            resourceId: message,
+            urlSigningSecretKey: secretKey,
+        });
+
+        const [version, expiresAt, hmac] = Buffer.from(signature, 'base64url').toString('utf8').split('.');
+        expect(signature).toBe('MC4wLjNUd2ZFRTY1OXVmU05zbVM0N2xS');
+        expect(version).toBe('0');
+        expect(expiresAt).toBe('0');
+        // This HMAC should match HMAC generated by Python function.
```

Comment:
From the perspective of a maintainer, it is strange to mention the Python counterpart here. Please either remove the comment or describe the code in a way that works without reading a repository in a different language.